src/components/FieldSleepMode/FieldSleepMode.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { RadioButton } from '../RadioButton';
+import PropTypes from 'prop-types';
+
+function FieldSleepMode({ onChange }) {
+  return (
+    <React.Fragment>
+      <RadioButton onChange={onChange} value="active" />
+      <RadioButton onChange={onChange} value="sleep" defaultChecked={true} />
+      <RadioButton onChange={onChange} value="fallingAsleep" />
```

Comment:
todo: You passed the value, but now the label is not correctly displayed. It should be "Falling Asleep" You could pass the label as `children`